cohorts/1/index.html

```diff
@@ -0,0 +1,38 @@
+<!DOCTYPE html>
+    <head>
+        <title>README</title>
+    </head>
+    <body>
+        <header>
+            <img src="assets/header.png" width="400" height="100"/>
+        </header>
+        <h1>Cohort Hypertext Assassins</h1>
+        <h3>Sparking joy one flame at a time \(^ヮ^)/</h3>
+        <h3>Cohort Members</h3>
+        <ul>
+            <li>
+                <a href="https://github.com/GTBitsOfGood/bootcamp/tree/fall-2020/hypertextAssassins-Emily">Emily Layton</a>
+            </li>
+            <li>
+                <a href="https://github.com/GTBitsOfGood/bootcamp/tree/fall-2020/hypertextAssassins-marlene">Marlene Garcia</a>
+            </li>
+            <li>
+                <a href="https://github.com/GTBitsOfGood/bootcamp/tree/fall-2020/hypertextAssassins-kirtana">Kirtana Mogili</a>
+            </li>
+            <li>
+                <a href="https://github.com/GTBitsOfGood/bootcamp/tree/fall-2020/hypertextAssassins-Eashan">Eashan Sinha</a>
+            </li>
+            <li>
+                <a href="https://github.com/GTBitsOfGood/bootcamp/tree/fall-2020/hypertextAssassins-carsen">Carsen Miller</a>
+            </li>
+            <li>
+                <a href="https://github.com/GTBitsOfGood/bootcamp/tree/fall-2020/hypertextAssassins-isha">Isha Palakurthy</a>
+            </li>
+        </ul>
+        <h3>Our Dear Mascot: Calcifer</h3>
+        <img src="https://media.giphy.com/media/qKmWi9mfycx6E/giphy.gif" width="250px" height="200px"/>
+        <h3>License</h3>
+        <p>This repository is licensed under the <a href="../../LICENSE.txt">GNU GPLv3 License</a>.</p>
+        <h3>Bits of Good Motto</h3>
```

Comment:
Again not sure if this should be here, but please correct me if I am wrong or disregard lmao